src/pad-utils.ts

```diff
@@ -0,0 +1,82 @@
+import { Namespace } from 'rdflib'
+import store from './store'
+import ns from './ns'
+
+const PAD = Namespace('http://www.w3.org/ns/pim/pad#')
+/**
+ * Get the chunks of the notepad
+ * They are stored in a RDF linked list
+ */
+
+// @ignore exporting this only for the unit test
+export function getChunks (subject, kb) {
+  const chunks: any[] = []
+  for (
+    let chunk = kb.the(subject, PAD('next'));
+    !chunk.sameTerm(subject);
+    chunk = kb.the(chunk, PAD('next'))
+  ) {
+    chunks.push(chunk)
+  }
+  return chunks
+}
+
+/**
+ *  Encode content to be put in XML or HTML elements
+ */
+// @ignore exporting this only for the unit test
+export function xmlEncode (str) {
+  return str.replace('&', '&amp;').replace('<', '&lt;').replace('>', '&gt;')
+}
+
+/**
+ * Convert a notepad to HTML
+ *   @param { } pad - the notepad
+ *   @param {store} pad - the data store
+ */
+export function notepadToHTML (pad: any, kb: store) {
```

Comment:
I am not 100% sure how to add this to index.ts?

Reply:
Can we go over this and also the participation after our standup?  I just have a couple of questions.